front/lib/data_sources.ts

```diff
@@ -72,3 +73,15 @@ export function canContainStructuredData(ds: DataSource): boolean {
         STRUCTURED_DATA_SOURCES.includes(ds.connectorProvider))
   );
 }
+
+export function canBeExpanded(
```

Comment:
`nit` but I'd keep this scope to data source view.

Reply:
You mean move it to another file ?

I picked data_source because it look at the data source, nothing specific to dataSourceView (from code usage)

Reply:
Yeah but we want to incentivise people to only deal (true in 90 % of the time) with only data source view.

Reply:
Actually we just don't have any data_source_views.ts file in `front/lib` so I guess it's fine here.